internal/server/iigointernal/judiciary.go

```diff
@@ -310,3 +338,75 @@ func removeSanctions(slice []roles.Sanction, s int) []roles.Sanction {
 func getDifferenceInLength(slice1 []roles.Sanction, slice2 []roles.Sanction) int {
 	return len(slice1) - len(slice2)
 }
+
+// pickUpRulesByVariable returns a list of rule_id's which are affected by certain variables.
+func pickUpRulesByVariable(variableName rules.VariableFieldName, ruleStore map[string]rules.RuleMatrix, variableMap map[rules.VariableFieldName]rules.VariableValuePair) ([]string, bool) {
```

Comment:
This function is found int rules package with a different signature. Is there a reason why you cant change the other version and use that instead. It seems like in this format there's going to be confusion as to which function to use at times

Reply:
@preetl Stole it from me and then pushed first XD. No you're right, I should use the rules version. Could I open this up as an issue for after this PR though, just because the whole rebasing goes haywire every time?